ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
         }
     }
 }
+
+extension BottomSheetViewController: UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        let shouldRecognizeSimultaneously = false
+        if let contentView = otherGestureRecognizer.view {
```

Comment:
we should be able to combine these if statements right?

Reply:
if let scrollView = otherGestureRecognizer.view as? UIScrollView, scrollView.contentOffset.y >= (scrollView.contentSize.height - scrollView.frame.size.height) || (scrollView.contentOffset.y <= 0) {

Reply:
i will update this